### PR TITLE
Update Synthetix Debt Pool adapter to reflect changes in SIP-165

### PR DESCRIPTION
**Pull request by cds95**
## Description

This PR updates the Synthetix debt pool adapter to reflect the changes in SIP-165.  The pseudocode for these changes are outlined here https://sips.synthetix.io/sips/sip-165/.  

## Steps to Test

1) Run adapter and send a request to it
2) Verify it successfully returns a response and verify that the hex values match with what is expected from the SIP.

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

**Review by cds95 — COMMENTED** [review]
block merging until release is done

**Review by austinborn — DISMISSED** [review]
LGTM

File: packages/sources/synthetix-debt-pool/src/endpoint/debt.ts [hunk 1]
```diff
@@ -72,6 +71,30 @@ const DEBT_POOL_ABI = [
...
+    name: 'totalNonSnxBackedDebt',
+    outputs: [
+      { internalType: 'uint256', name: 'excludedDebt', type: 'uint256' },
+      { internalType: 'bool', name: 'isInvalid', type: 'bool' },
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function',
+  },
+]
```
Comment (austinborn) [inline packages/sources/synthetix-debt-pool/src/endpoint/debt.ts:86]: Might be nice to include the source for these ABIs as well

Comment (cds95) [reply packages/sources/synthetix-debt-pool/src/endpoint/debt.ts:86]: what's the source?

Comment (austinborn) [reply packages/sources/synthetix-debt-pool/src/endpoint/debt.ts:86]: I was thinking either the address of the deployed instance of the contract so people can check it on etherscan, or if synthetix has reference documentation somewhere...

**Review by austinborn — APPROVED** [review]

**Pull request merged**